pages/posts/[slug].tsx

```diff
@@ -0,0 +1,53 @@
+import AuthorCard from '../../components/AuthorCard';
```

Comment:
Formatting preference: `import` not referencing to `node_modules` should be below those which referenced to `node_modules`.